flags.css

```diff
@@ -8,28 +8,41 @@
    height: 50%;
 }
 .red {
-  background-color: #f00;
+  background-color: rgb(255, 0, 0);
 }
 .white {
-  background-color: #f5f5f5;
+  background-color: rgb(245, 245, 245);
 }
 /* Colombia */
-.quarterRectangle {
-  height: 25%;
-}
 .mustardYellow{
-  background-color: #fcd116;
+  background-color: rgb(252, 209, 22);
 }
 
 .darkBlue {
-  background-color: #003893;
+  background-color: rgb(0, 56, 147);
 }
 
 .darkRed {
-  background-color: #CE1126;
+  background-color: rgb(206, 17, 38);
 }
 
 /* Ireland */
+.flexContainer {
+  display: flex;
+  flex-direction: row;
+}
+
+.flexChild {
+  width: 100%;
```

Comment:
this feels like a bad use of a class name.

Because really the point of doing it this way is to make the various elements to shrink or grow based on siblings.

Maybe dynamicSiblingPanel?

Reply:
you could have also specified 33% and it would have worked for this.  not a big deal, just fyi